sound/soc/sof/intel/mtl.c

```diff
@@ -771,6 +771,8 @@ int sof_mtl_ops_init(struct snd_sof_dev *sdev)
 	ipc4_data = sdev->private;
 	ipc4_data->manifest_fw_hdr_offset = SOF_MAN4_FW_HDR_OFFSET;
 
+	ipc4_data->mtrace_type = SOF_IPC4_MTRACE_INTEL_CAVS2X;
```

Comment:
Its awekward to call it MTRACE_INTEL_CAVS2X for MTL. Can we find a better name that applies to both CAVS2X and ACE?

Reply:
indeed this really confusing naming. MTL is NOT CAVS2x

Reply:
Agree with @ujfalusi . Other option is to use arbitrary versions, but those would be a version defined only in Linux driver, so I'd say potential for even more confusion. The "platform where this was introduced" naming concept seems least bad.

Reply:
why can't we list all known types of MTRACE today, and only add support for what we tested. Adding definitions like this seems very ad-hoc and not based on architecture/firmware evolutions.

Reply:
If we don't need to document the differences, then OK and there is this fourth version I have seen bugs me a lot.